fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/MetricLoader.java

```diff
@@ -18,7 +21,12 @@
      * @deprecated in favor of loadMetricDictionary(MetricDictionary, DimensionDictionary)
      */
     @Deprecated
-    void loadMetricDictionary(MetricDictionary metricDictionary);
+    default void loadMetricDictionary(MetricDictionary metricDictionary) {
+        String message = "The method loadMetricDictionary(MetricDictionary, DimensionDictionary) must be implemented " +
+                "for MetricLoader";
+        LoggerFactory.getLogger(DimensionToNameSerializer.class).error(message);
```

Comment:
I was actually suggesting that while some metric loaders depend on filtered metrics and thus dimension dictionaries, not everyone would. 
But I don't care that much, we can ask them to do so via deprecation.